CRM/Report/Form/Campaign/SurveyDetails.php

```diff
@@ -182,6 +182,11 @@ public function __construct() {
             'required' => TRUE,
             'title' => ts('Survey Result'),
           ),
+          'activity_date_time' => array(
+            'name' => 'activity_date_time',
+            'title' => ts('Date'),
+            'type' => CRM_Utils_Type::T_DATE,
```

Comment:
This isn't quite right - you can't find activities on the last day of the filtered period (see below).  I tried fixing this by adding `CRM_Utils_Type::T_TIME` (which I've used to fix this bug [before](https://github.com/eileenmcnaughton/nz.co.fuzion.extendedreport/pull/57)), but it doesn't seem to have the desired effect here.